include/dem/data_containers.h

```diff
@@ -207,6 +207,12 @@ namespace DEM
                 typename Triangulation<dim - 1, dim>::active_cell_iterator>>>
       floating_mesh_information;
 
+    // [[(cell iterator, cell iterator)]]
+    typedef std::vector<std::vector<
```

Comment:
Here the first of the pair is the background grid cell, and the second of the pair is from the solid object? I would specifiy if in a comment